src/DataTable/doc/index.rst

```diff
@@ -0,0 +1,115 @@
+and create a table ::
+
+Minimalistic example::
+
+        $table = $tableBuilder
+                ->addData([
+                        ['id' => 1, 'pseudo' => 'Bob'],
+                        ['id' => 2, 'pseudo' => 'Kitty'],
+                    ])
+                    ->addOptions([
+                        'paging' => false,
+                        'searchable' => false,
+                    ])
+                ->createTable();
+
+A more complete example from a controller::
+
+    #[Route('/usersList', name: 'app_users_list')]
+    public function index(TableBuilderInterface $tableBuilder, UserRepository $userRepository): Response
+    {
+        $manifestPath = $this->getParameter('kernel.project_dir').'/public/build/manifest.json';
+        $package = new Package(new JsonManifestVersionStrategy($manifestPath));
+
+        $users = $userRepository->getUsersWithPostsAndCommentsCount();
+        $table = $tableBuilder
+                    ->addData($users)
+                    ->addOptions([
+                        'paging' => true,
+                        'perPage' => 10,
+                        'searchable' => true,
+                        // Optionally import Simple-DataTables CSS file from CDN
+                        'withPluginCss' => true,
+                        // Optionally add custom CSS file, here through webpack entry and manifest.json
+                        'withCss' => $package->getUrl('build/datatable.css'),
+                        // Customize labels (@see https://github.com/fiduswriter/Simple-DataTables/wiki/labels)
+                        'labels' => [
+                            'placeholder' => 'Find a customer...',
+                            'perPage' => 'Show {select} customers per page',
+                        ],
+                        // Customize layout (@see https://github.com/fiduswriter/Simple-DataTables/wiki/layout)
+                        'layout' => [
+                            'top' => '{search}{select}',
+                            'bottom' => '{info}{pager}',
+                        ],
+                        // Customize columns (@see
+                        'columns' => [
+                            ['select' => 3, 'sort' => 'desc'],
+                        ],
+                        // Useful accessibility options
+                        'rowNavigation' => true,
+                        'tabIndex' => 1,
+                        // ... and more
+                    ])
+                    ->createTable();
+
+        return $this->render('datatable/index.html.twig', [
+            'table' => $table,
+        ]);
+    }
+
+You can pass most of the options supported by the `Simple-DataTables`_ library.
+Check out the `Simple-DataTables documentation`_  to discover them all.
+
+Once created in PHP, a table can be displayed using Twig if installed
+(requires `Symfony Webpack Encore`_):
+
+.. code-block:: twig
+
+    {{ render_table(table) }}
+
```

Comment:
what about include the demo gif here?
or perhaps it would be better to showcase on ux.symfony.com directly in this PR [updating here](https://github.com/symfony/ux/tree/2.x/ux.symfony.com)